lib/liquid/combinators/tags/capture.ex

```diff
@@ -13,6 +13,13 @@ defmodule Liquid.Combinators.Tags.Capture do
   import NimbleParsec
   alias Liquid.Combinators.Tag
 
+  @type t :: [
+          capture: [
+            variable_name: String.t(),
+            parts: String.t() | Liquid.NimbleParsec.t()
```

Comment:
`Liquid.t()` includes `String.t()` (see the next comments) so `parts: Liquid.t()`